src/smc-webapp/frame-editors/terminal-editor/connected-terminal.ts

```diff
@@ -118,8 +118,12 @@ export class Terminal {
     this.set_connection_status("disconnected");
   }
 
+  static get rendererType() {
+    return "dom";
+  }
+
   private get_xtermjs_options(): any {
-    const rendererType = "dom";
+    const rendererType = Terminal.rendererType;
```

Comment:
Why did you change this to a computed getter?   I can't see any gain, except to make things more complicated and harder to understand.  Also, the code is now inconsistent in style to the code for `const fontFamily = "monospace";` directly below.   

The only other renderer is canvas, and it is unusably slow to create new Terminals, so we will likely never use it.

Reply:
Can you address whether this fix should really be upstream (in your opinion)?

Reply:
well, I just wanted this to be read-only attribute... there is no `const this.attr = ...`